.gitignore

```diff
@@ -38,4 +38,5 @@ jspm_packages
 .node_repl_history
 
 dist
-sessions
\ No newline at end of file
+sessions
+.directory
```

Comment:
is `.directory` a file extension or just a folder? if it's a folder just `directory` does it.

Also new line at the end